libsplinter/src/peer/interconnect.rs

```diff
@@ -107,6 +117,16 @@ impl ShutdownHandle for PeerInterconnect {
     /// For components with threads, this should break out of any loops and ready the threads for
     /// being joined.
     fn signal_shutdown(&mut self) {
+        self.pacemaker_shutdown_signaler.shutdown();
+
+        if self
+            .pending_shutdown_sender
+            .send(RetryIncoming::Shutdown)
+            .is_err()
+        {
+            warn!("Peer Interconnect is no longer running");
```

Comment:
This should probably have more context because this statement in the log context won't have much meaning.

Reply:
added more context